### ISSUE TITLE
[Feature] Is gvfs an optional dependency?

### ISSUE THREAD

**Issue by miyl** [ErikReider/SwayNotificationCenter#439]
Trying to install this on Arch, and  partially because of the `gvfs` dependency it needs to pull in all sorts of indirect dependencies like `udisks`, which in turn brings in things like `parted`. To me installing a partitioning tool seems a bit unnecessary just to get notifications.
When I read #144 it seems `gvfs` might just be a dependency for "MPRIS covers", but it seems from the README that MPRIS itself is just an "available widget" which can be removed. Does this make `gvfs` an optional dependency?
If so, are there perhaps other dependencies that are optional dependencies?

> sudo pacman -S swaync
> resolving dependencies...
> looking for conflicting packages...
> Packages (21) gcr-4-4.3.0-1  granite-6.2.0-2  gvfs-1.54.1-1  libatasmart-0.19-6
>               libblockdev-3.1.1-2  libblockdev-crypto-3.1.1-2  libblockdev-fs-3.1.1-2
>               libblockdev-loop-3.1.1-2  libblockdev-mdraid-3.1.1-2
>               libblockdev-nvme-3.1.1-2  libblockdev-part-3.1.1-2
>               libblockdev-swap-3.1.1-2  libbytesize-2.8-3  libgee-0.20.6-1
>               libhandy-1.8.3-1  libnvme-1.9-1  mdadm-4.3-2  parted-3.6-1
>               udisks2-2.10.1-4  volume_key-0.3.12-9  swaync-0.10.1-2
> 
> Total Download Size:    2,68 MiB
> Total Installed Size:  38,26 MiB

**Comment by notquitethereyet** [comment]
I think you don't need gvfs. I only had to install it to get album art in the control center

**Comment by ErikReider** [comment]
Yeah, probably wouldn't be needed if we switch to GTK 4 (which is a painful transition specifically for swaync) due to GTK 4 deprecating the use of Pixbuf
